openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -0,0 +1,55 @@
+"""
+Command to load course overviews.
+"""
+import logging
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
I didn't see tests for other management commands and thus didn't write any here--thought it wasn't standard.  Do you think it's needed here?